src/Features/CSharp/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1 +0,0 @@
-
```

Comment:
that's me. made a change and backed out, but some whitespace differences stuck. I likely need to scrub some commits and rewrite history.